views/artist-search-results.hbs

```diff
@@ -0,0 +1,6 @@
+{{#each allArtists}}
+<div class=‘image’>
+  <p>{{this.name}}</p>
+  <a href="/albums/someArtistIdGoesHere">View Albums</a>
```

Comment:
Here you can not pass the ID hard coded you need to make it dynamic, the same way you did for the name:

```
<a href="/albums/{{this.id}}">View Albums</a>
```